packages/orbit-components/src/Loading/consts.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 const TYPE_OPTIONS = {
-  BUTTON_LOADER: "buttonLoader",
-  SEARCH_LOADER: "searchLoader",
   BOX_LOADER: "boxLoader",
-  PAGE_LOADER: "pageLoader",
+  SEARCH_LOADER: "searchLoader",
   INLINE_LOADER: "inlineLoader",
+  BUTTON_LOADER: "buttonLoader",
+  PAGE_LOADER: "pageLoader",
```

Comment:
It seems like you just reordered these, right? 🤔

Reply:
yes, because first i did something like ```Object.values(TYPE_OPTIONS).slice(0, 3)```, later rewrite it in a different way but that change remained :)